recipes/chunky3d/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "chunky3d" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 768037c8b24977e898a884564cc1447c4a125ba058ffa048e78ab0ca7c9044a5
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - dill
+    - itk
+    - msgpack-numpy
+    - msgpack-python
+    - networkx
+    - numba
+    - psutil
+    - scipy
+    - simpleitk
+    - vtk
+    
+
+test:
+  imports: []
+  # TODO: make `test` a package, then probably:
+  #  - test.test_sparse
+  #  - test.test_sparse_func
```

Comment:
you should have some kind of real test

Reply:
I don't fully understand the `run_tests.py` - should I write it myself and include in the recipe, or is it auto-generated and I need only to specify the modules (containing tests) which will be imported?

If the tests need to be importable, I need to make the test/ subdirectory of my sdist package a module (it's a raw folder now). For that I probably need to publish a next version on PyPI.

Or maybe - the tests are running in the unpacked source directory, and I just need to write `command: "{{ PYTHON }} test/test_sparse.py && {{ PYTHON }} test/test_sparse_func.py`?

Reply:
OK, I found this: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section
I think I'm on my way to fixing this.

Reply:
Also you can look at the exampple or any of the feedstock repos in this org